src/_guides/language/concurrency/index.md

```diff
@@ -295,6 +295,12 @@ The worker isolate can block without affecting other isolates.
 
 [`send()` method]: {{site.dart-api}}/{{site.data.pkg-vers.SDK.channel}}/dart-isolate/SendPort/send.html
 
+Message passing normally involves data copying, and thus can be slow.
+However, when you send data using isolates,
+the memory that holds the message in the worker isolate isn’t copied,
+but instead is transferred to the receiving isolate.
```

Comment:
This is not generally true (and not sure what "send data using isolates" mean). It's only true for the message sent using `Isolate.exit`.
So, should it say: "When you send data using `Isolate.exit`,"?

Reply:
That's correct, this paragraph was previously under the first example and specifically about `SendPort.send` vs. `Isolate.exit()`. I was trying to generalize it and make it a conceptual summary instead of the specific mention of those methods because they're abstracted now. But, I don't want to completely remove the info...

I'll see how I can incorporate the valuable parts of this more accurately and then request your review again, thanks!